twoops_tracker/py/twoopstracker/health.py

```diff
@@ -10,7 +10,10 @@ def health_check(request):
     # 1. has there been any tweets collected in the last 24 hours\
     # and if not it might indicate the system not being functional
     try:
-        Tweet.objects.count()
-        return Response(status=status.HTTP_200_OK)
+        first_tweet = Tweet.objects.only("tweet_id").first()
+        if first_tweet:
+            return Response(status=status.HTTP_200_OK)
```

Comment:
Don't we want to return the tweet_id on success?

Reply:
@kilemensi I don't think we have to return the `tweet_id` because we are not consuming it anywhere. Looking at our uptime configuration, I believe our measure of an `up/down status` is based on a response time of `< 30 seconds`, (the default)

Reply:
We don't **_have to_** return anything @thepsalmist. I was talking more from good API design standpoint.

1. It's a `GET` request so it's good to return _something_, and
2. The returned tweet id can help offer insight/verify that indeed all is well with the system i.e. new tweets are being pulled in and are accessible to users.